python-crud-google-spreadsheet/src/create.py

```diff
@@ -0,0 +1,18 @@
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+import os
+
+
+def handler(body, req):
+    if (body is None or body['values'] is None or
+            type(body['values']) is not list):
```

Comment:
Consider using `instanceof`. You might be getting a subclass of a list for some reason.

Reply:
I'm not sure the a list subclass would cause an issue, but I'll change it just in case. Addressing the comment about empty lists and other cases, this would limit it to being a list, which is the only requirement I have set for using it. There is no reason that the row shouldn't be empty.